grpc/server.go

```diff
@@ -123,7 +86,21 @@ func NewServer(c *cmd.GRPCServerConfig, tlsConfig *tls.Config, statsRegistry pro
 				MaxConnectionAge: c.MaxConnectionAge.Duration,
 			}))
 	}
-	return grpc.NewServer(options...), l, nil
+
+	server := grpc.NewServer(options...)
+
+	healthServer := health.NewServer()
+	healthpb.RegisterHealthServer(server, healthServer)
+
+	start := func() error {
+		return cmd.FilterShutdownErrors(server.Serve(listener))
```

Comment:
I think we can move FilterShutdownErrors into grpc now. We probably could have before but there was one lingering use in boulder-va.